app/controllers/admin/movies_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
   end
 
   def show
+    @youtube_id = @movie.trailer.split("=").last
```

Comment:
whats going on here?
kindly elaborate

Reply:
Getting the youtube id for the video from the link provided by user. As it contains id after "=".